groovy-language-server/src/test/java/com/palantir/groovylanguageserver/GroovyTextDocumentServiceTest.java

```diff
@@ -162,4 +200,14 @@ public void testDocumentSymbols_relativePath() throws InterruptedException, Exec
                 is(symbolsMap.get(WORKSPACE_PATH.resolve("something.groovy").toString())));
     }
 
+    @Test
+    public void testReferences() throws InterruptedException, ExecutionException {
+        ReferenceParams params =
+                (ReferenceParams) new ReferenceParamsBuilder().context(false).position(5, 5).textDocument("uri")
```

Comment:
sigh looks like a `ReferenceParamsBuilder` bug and the calls on setting `TextDocumentPositionParams` properties will make it return a TextDocumentPositionParamsBuilder...